knotx-repository-connector/knotx-repository-connector-http/src/main/java/io/knotx/repository/impl/HttpRepositoryConnectorProxyImpl.java

```diff
@@ -138,16 +138,16 @@ private String buildRepoUri(ClientRequest repoRequest) {
     if (params != null && params.names() != null && !params.names().isEmpty()) {
       uri.append("?")
           .append(params.names().stream()
-              .map(name -> new StringBuilder(name).append("=")
-                  .append(encodeParamValue(params.get(name))))
+              .map(name -> new StringBuilder(encode(name)).append("=")
```

Comment:
Please write the Unit test for this case.

Reply:
Test added, but as a part of `SampleApplicationTest`.